test/synthetics/latest_release_cdn.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * API Synthetic check based on https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/scripting-monitors/write-synthetic-api-tests/
+ * Flow:
+ *   1. Get latest release tag from Github for infra-agent and integrations.
+ *   2. Validate the right package is published in download server.
+ * 
+ * How to install:
+ *   1. Go to Synthetics > Create monitor in NR1.
+ *   2. Select Scripted API check.
+ *   3. Define appropiate settings and copy/paste this script as last step.
+ *   4. Optionally, create an Alert Condition to get notifications.
+ */
+var assert = require('assert');
+var downloadServerBaseURI = `https://download.newrelic.com/infrastructure_agent`;
+var githubOrg = `https://api.github.com/repos/newrelic`;
+var allRepos = ['infrastructure-agent','nri-jmx', 'nrjmx', 'nri-snmp'];
+/* 
+ * Other OHIs: to consider in a separate synthetic check and alert
+'nri-mysql', 'nri-apache', 'nri-cassandra', 'nri-consul', 'nri-couchbase', 
+'nri-elasticsearch', 'nri-f5', 'nri-haproxy', 'nri-kafka', 'nri-memcached', 
+'nri-mongodb', 'nri-mssql', 'nri-mysql', 'nri-nagios', 'nri-nginx', 'nri-oracledb', 
+'nri-postgresql', 'nri-rabbitmq', 'nri-redis'];
+*/
+var arm64Unsupported = ['nri-oracledb'];
+var windowsUnsupported = ['nri-oracledb'];
+
+function checkLatestReleases(repos){
+  repos.forEach(repo => {
+      console.log(`Getting ${githubOrg}/${repo}/releases/latest`);
+      $http.get({
+        'uri': `${githubOrg}/${repo}/releases/latest`,
+        'headers': {'User-Agent': 'newrelic-ohai'}
+      },
+        function (err, response, body) {
+          assert.equal(response.statusCode, 200, `Expected 200 getting releases got ${response.statusCode} ${err}`);
```

Comment:
will this assert exit the script or will it continue?

Reply:
yes, I've seen a failed assert stopping the script.